runtime/queries/rescript/textobjects.scm

```diff
@@ -1,16 +1,114 @@
+; Queries for nvim-treesitter/nvim-treesitter-textobjects
```

Comment:
This line could be removed or could point to the upstream source for these queries if there is one